_solutions/automations/calculate-week-range.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Calculate week ranges
+excerpt: Use `date_lerp` to calculate the date ranges for a year.
+summary: 
+layout: solution
+permalink: /solutions/automations/calculate-week-range
```

Comment:
You don't need a `permalink` for any of the solutions hub pages. They're based on the path and file name automatically.